### PR TITLE
Deployment upgrade

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-290.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-290.dev.beta.nhschoices.net)

File: Dockerfile [hunk 1]
```diff
@@ -27,4 +27,4 @@ USER $USERNAME
 
 RUN [ "yarn", "run", "build-css" ]
 
-CMD [ "yarn", "start" ]
+CMD [ "node", "app.js" ]
```
Comment (st3v3nhunt) [inline Dockerfile]: Can a comment be added here to explain why this is as it is so nobody tries to change it back to using Yarn to run something? Thanks!

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-290.dev.beta.nhschoices.net)

**Comment by st3v3nhunt** [comment]
@adamgraves-choices #291 should sort out the broken build

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-290.dev.beta.nhschoices.net)

**Comment by st3v3nhunt** [comment]
@adamgraves-choices Noticed a typo in `--pure-lockfile` flag. PR https://github.com/nhsuk/connecting-to-services/pull/292

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-290.dev.beta.nhschoices.net)

**Pull request merged**